ironfish/src/mining/manager.ts

```diff
@@ -380,15 +379,6 @@ export class MiningManager {
       }
     }
 
-    const validation = await this.node.chain.verifier.verifyBlock(block)
```

Comment:
Looks like the `verifyBlock` does more verification than the verify logic in `addBlock`

`addBlock` only verify the block header, but `verifyBlock` also verify the transaction proof and miner fee

Reply:
After reading the code, `addForkToChain` also calls `verifyBlock` eventually. Thanks for finding this!